Pipfile

```diff
@@ -73,3 +73,4 @@ python_version = "3.6"
 test = "python setup.py test"
 build_sphinx = "python setup.py build_sphinx"
 dev = "pytest --no-cov -vs tests"
+setup = "./scripts/setup"
```

Comment:
`PIPENV_QUIET=1 ./script/setup` instead or better include `PIPENV_QUIET=1` in the ./scripts/setup?

Reply:
`PIPENV_QUIET=1` have an impact on the pipenv command only if used outside the pipenv itself. So before the `pipenv run setup` or before the script IF it includes `pyenv shell` inside.

It is to avoid the *Courtesy note* (Cf. https://github.com/pypa/pipenv/issues/3068). But our pipenv version have a bug on it.

Reply:
@jma: `pyenv shell` seems to need more preparation before being usable for this.

```
❯ ./scripts/setup
pyenv: shell integration not enabled. Run `pyenv init' for instructions.
❯ pyenv init
# Load pyenv automatically by appending
# the following to ~/.zshrc:

eval "$(pyenv init -)"
```

It remains more comfortable to use the [custom shortcut functionnality of pipenv](https://pipenv.readthedocs.io/en/latest/advanced/#custom-script-shortcuts): 

  * no more preparation for the developer
  * a message explains to developer which command to use